test/behavesLikeSqlFormatter.ts

```diff
@@ -195,18 +195,18 @@ export default function behavesLikeSqlFormatter(format: FormatFn) {
   });
 
   it('keeps short parenthesized list with nested parenthesis on single line', () => {
-    const result = format('SELECT (a + b * (c - NOW()));');
+    const result = format('SELECT (a + b * (c - SIN(1)));');
     expect(result).toBe(dedent`
       SELECT
-        (a + b * (c - NOW()));
+        (a + b * (c - SIN(1)));
     `);
   });
 
   it('breaks long parenthesized lists to multiple lines', () => {
     const result = format(`
       INSERT INTO some_table (id_product, id_shop, id_currency, id_country, id_registration) (
-      SELECT IF(dq.id_discounter_shopping = 2, dq.value, dq.value / 100),
-      IF (dq.id_discounter_shopping = 2, 'amount', 'percentage') FROM foo);
+      SELECT COALESCE(dq.id_discounter_shopping = 2, dq.value, dq.value / 100),
+      COALESCE (dq.id_discounter_shopping = 2, 'amount', 'percentage') FROM foo);
```

Comment:
Switched from `IF` to `COALESCE` which is supported in all dialects.

Reply:
interesting that IF isn't as universal as COALESCE